testdata/expected/docker/java-android-gradle-sample/default-group/java-android-gradle-sample/JavaArtifact.graph.json

```diff
@@ -1,1241 +0,0 @@
-{
```

Comment:
Let's go ahead and just generate expected output for Android in Docker. That will make it so srclib-core doesn't need to handle the case where test cases exist for program mode but not docker mode. Ideally, the 2 should be as identical as possible.